src/platform/windows/drop_handler.rs

```diff
@@ -77,36 +82,49 @@ impl FileDropHandler {
         pDataObj: *const IDataObject,
         _grfKeyState: DWORD,
         _pt: *const POINTL,
-        _pdwEffect: *mut DWORD,
+        pdwEffect: *mut DWORD,
     ) -> HRESULT {
         use events::WindowEvent::HoveredFile;
+        use winapi::um::oleidl::{DROPEFFECT_NONE, DROPEFFECT_COPY};
```

Comment:
Alphabetized:
```suggestion
        use winapi::um::oleidl::{DROPEFFECT_COPY, DROPEFFECT_NONE};
```
I also think it would be fine to import these at the module-level, just to have less noise in the functions. Though, if you have a preference for function-level imports, it's perfectly okay to keep them here.